lib/grammers-client/src/types/input_message.rs

```diff
@@ -46,12 +51,68 @@ impl InputMessage {
         self
     }
 
+    /// Replaces the plaintext in the message.
+    ///
+    /// <div class="warning">
+    /// Note that this method does not modify formatting entities, which may break
+    /// formatting or cause out-of-bounds errors if entities does not match the given
+    /// text.
+    /// <br><br>
+    ///
+    /// The caller must ensure that formatting entities remain valid for the given text.
+    /// If you need to update formatting entities, call method
+    /// [`InputMessage::fmt_entities`].
```

Comment:
I also think this does not need to be part of the warning (for both here and the media).

Reply:
Fixed